notebooks/en/rag_with_knowledge_graphs_neo4j.ipynb

```diff
@@ -0,0 +1,644 @@
+{
```

Comment:
<p>"Let's find out which researcher has the most peer collaborations."</p><p> </p><p>"Here, we need to start from..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>